docker-compose.yml

```diff
@@ -6,11 +6,14 @@ services:
     build: .
     image: fossmarks
     restart: always
-    environment:
-      - VIRTUAL_HOST=fossmarks.org,www.fossmarks.org
-      - VIRTUAL_PORT=8080
-      - LETSENCRYPT_HOST=fossmarks.org,www.fossmarks.org
-      - LETSENCRYPT_EMAIL=contact@fsfe.org
```

Comment:
Please leave those in. We are still in a strange state where we want fossmarks.org to point to cont2 where we currently don't have the caddy setup.